ui/admin/app/routes/scopes/scope/aliases/alias.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class ScopesScopeAliasesAliasRoute extends Route {
+  @service store;
+  @service can;
```

Comment:
None blocking comment: Despite we are not using right now the service `can`, I do understand we add it because later we will add permissions and therefore service `can` is necessary?

Reply:
Actually, the 'can' may not be necessary at all. I'm looking at other similar routes where we didn't need it either. I'm surprised the linter did not catch this, but it will be needed in the '/new' routes. I'll remove this, ty for catching this

Reply:
You are right, the linter should help us with this.

Since this is an specific ember thing the "default" eslint will not catch it as something we declared and we did not use. But looking at the `eslint-plugin-ember` I found this rule [no-unused-services](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-unused-services.md) it is not part of the default rules so if we want it we have to activate it. I will bring this topic to tech time and we can discuss as a team 😉

Reply:
yup, that's a good idea! removed it